templates/odoo/0/docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 odoo:
   image: odoo
   ports:
-    - "8069:8069"
+    - ${port}:8069
   links:
     - db
 db:
   image: postgres
   environment:
     - POSTGRES_USER=odoo
-    - POSTGRES_PASSWORD=odoo
+    - POSTGRES_PASSWORD=${pgsql_password}
```

Comment:
You also need to add questions for `${port}` and `${pgsql_password}` at rancher-compose.yml file. Why not to add also `${pgsql_user}`??